.github/workflows/CI.yml

```diff
@@ -40,30 +41,45 @@ jobs:
       
       - name: tar
         run: tar zcvf sqlite_fdw.tar.gz ./*
-      
+
       - name: set_proxy
         run: bash GitHubActions/env.sh
 
       - name: install locales
         run: bash GitHubActions/install_locales.sh
-            
-      - name: build PostgreSQL ${{ matrix.pg }}
-        run: bash GitHubActions/build_postgres.sh ${{ matrix.pg }}
 
-      - name: install SQLite
-        run: bash GitHubActions/install_sqlite.sh ${{ env.SQLITE_VERSION }} ${{ env.SQLITE_YEAR }}
-      
-      - name: build sqlite_fdw
-        run: bash GitHubActions/build_sqlite_fdw.sh ${{ matrix.pg }}
-      
+      - name: build PostgreSQL ${{ matrix.pg }} with ${{ matrix.config }}
+        run: |
+          if [[ "${{ matrix.config }}" == "default" ]]; then
+            bash GitHubActions/build_postgres.sh ${{ matrix.pg }}
+          elif [[ "${{ matrix.config }}" == "new_config" ]]; then
+            bash GitHubActions/build_postgres.sh ${{ matrix.pg }} --enable-cassert --enable-debug
+          fi
+
+      - name: install SQLite with ${{ matrix.postgis }}
```

Comment:
It seems  that "matrix.postgis" is not defined anywhere.

Reply:
It's wrong. matrix.config not matrix.postgis. I updated new version.

Reply:
Thank you for fixing.